db/academyPortal.sql

```diff
@@ -21,446 +21,57 @@ SET NAMES utf8mb4;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
 
-# Dump of table applicants
```

Comment:
I think you've deleted all of the tables except applicants_additional.  Could you please add the other tables back in?